ocpp/charge_point.py

```diff
@@ -248,18 +248,16 @@ def __init__(self, id, connection, response_timeout=30):
         # Use a lock to prevent make sure that only 1 message can be send at a
         # a time.
         await self._call_lock.acquire()
```

Comment:
If you used `async with self._call_lock:`, you wouldn't need to worry about releasing the lock. In my opinion, it's a bit cleaner as well.

Reply:
Thanks for the suggestion! I used it in #51

Reply:
Thanks for the suggestion. I used it in #51